docs/sources/mimir/operators-guide/run-production-environment/production-tips/index.md

```diff
@@ -24,13 +26,13 @@ We recommend fine-tuning the following settings to avoid reaching the maximum nu
 1. Configure the system's `file-max` ulimit to at least `65536`. Increase the limit to `1048576` when running a Grafana Mimir cluster with more than a thousand tenants.
 1. Enable ingesters [shuffle sharding]({{< relref "../../../configure/configure-shuffle-sharding/index.md" >}}) to reduce the number of tenants per ingester.
 
-### Ingester disk space
+### Ingester disk space requirements
 
-The ingester writes received samples to a write-ahead log (WAL) and by default, compacts them into a new block every two hours.
-Both the WAL and blocks are temporarily stored on the local disk.
+The ingester writes received samples to a write-ahead log (WAL) and by default, compacts them into a new block every configured time interval.
```

Comment:
We don't document the ability to change this anymore because it often leads to problems when users aren't intimately familiar with Mimir internals. So we should just leave the "two hours"

Reply:
This comment was marked as resolved, but I agree with Nick and rolled back to "not resolved". I think we should get stick to 2 hours in our documentation. Changing it is a controversial topic and we don't want customers / OSS community to touch it. We also consider to remove that option at all in the future (we discussed it few times).